cspyce/__init__.py

```diff
@@ -1,9 +1,9 @@
 ################################################################################
 # cspyce/__init__.py
 ################################################################################
-# SPYCE OVERVIEW
+# CSPYCE OVERVIEW
 #
-# Spyce is a Python module that provides an interface to the CSPICE library. It
+# Cspyce is a Python module that provides an interface to the CSPICE library. It
```

Comment:
cspyce should be lower case

Reply:
I have no opinion on this. Change it if you feel strongly.